integration-tests/drivers/TestDriver066/package-lock.json

```diff
@@ -2119,9 +2119,9 @@
       }
     },
     "node_modules/@heap/react-native-heap": {
-      "version": "0.19.0",
-      "resolved": "file:../../heap-react-native-heap-0.19.0.tgz",
-      "integrity": "sha512-GTeA2RuqmBEuCbX31NbGLiAEo3f7hj0ZjFfxWjiqkmKKyYS4pXENYhd0uQlxV+1K+PlymhR95tvE4t0L1jQJ3g==",
+      "version": "0.20.0",
```

Comment:
This is a much more reasonable change size now that we're not reformatting this whole file. 😄